src/Components/Form/Form.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC, useState } from 'react'
+import './Form.css'
+interface SearchCityProps {
+  onSearch: (cityInput: string) => void;
+}
+
+export const Form: FC<SearchCityProps> = ({onSearch}) => {
```

Comment:
We might want to consider making all exports or export default besides having a mix of the two. Just to make things more consistent.